tests/unit-tests/reports/overview/list-table/test-class-sensei-reports-overview-list-table-lessons.php

```diff
@@ -99,7 +99,7 @@ public function testGetColumns_WhenCalled_ReturnsColumnsWithMatchingTotals() {
 			wp_update_comment(
 				[
 					'comment_ID'   => $lesson_activity_comment_id,
-					'comment_date' => gmdate( 'Y-m-d H:i:s', strtotime( ( $days_count * 24 ) - 6 . ' hours' ) ),
+					'comment_date' => gmdate( 'Y-m-d H:i:s', strtotime( ( $days_count * 24 ) . ' hours' ) ),
```

Comment:
Just a question, why did we remove this here?

Reply:
Some GMT issue, causes the test to be a bit flaky. Removed for safety